### DISCUSSION TITLE
Need suggestion for motor control application with different ON/OFF switch

### DISCUSSION THREAD

**Discussion by u3dhaya** [arendst/Tasmota#12881] [Q&A] [answered]
Hi, 
I am completely new for Tasmota with very basic knowledge of this platform. I found tasmota recently and successfully designed a project for simple smart switch through Alexa voice control.

Now I want to control a water pump motor which has a separate ON(green) & Off (red) switch. Both the switches are need momentary action. The function I need will be as follows,

1. Relay1 need to turn ON for one second 
2. Timer for 10 minutes
3. Relay2 should turn ON for one second after 10 minutes time delay
4. end of cycle

Is it possible to do this with Tasmota. Any suggestion will be a big helpful for me...

Udhay

**Comment by TrDA-hab** [comment, marked as answer]
https://tasmota.github.io/docs/Rules/
https://templates.blakadder.com/sonoff_DUALR3.html

**Reply by sfromis** [reply]
Using the PulseTime command setting would be very useful for the one-second "on" pulses, and check the RuleTimer command for timing within rules.
